angular/src/app/shared/data-table/data-table.component.html

```diff
@@ -13,3 +14,9 @@
                  [multiSortKey]="defaultTableConfig.multiSortKey"
                  [navigateToNextCell]="navigateToNextCell">
 </ag-grid-angular>
+<div style="align-self: flex-end;">
+  <button (click)="onBtnExportDataAsCsv()" mat-stroked-button color="accent">
+    Download as CSV
+  </button>
```

Comment:
1. According to the design, it is an icon button (`mat-icon-button`).
2. The name of the icon in question should be found in https://material.io/resources/icons/?style=baseline.